doc/source/contributor-ref-good-first-contributions.rst

```diff
@@ -37,6 +40,4 @@ We wish we had more time to write usage examples because we believe they help
 users to get started with building what they want to build. Here are a few
 ideas where we'd be happy to accept a PR:
 
-- Llama 2 fine-tuning, with Hugging Face Transformers and PyTorch
-- XGBoost
 - Android ONNX on-device training
```

Comment:
Do we want Android ONNX on-device for now?

Reply:
If external contributors are willing to work on it, I would say yes!

Reply:
I removed it anyways as it was the only element in the list ^^